src/edu/stanford/dlss/was/WasapiDownloader.java

```diff
@@ -1,8 +1,49 @@
 package edu.stanford.dlss.was;
 
+import java.io.*;
+import java.util.Properties;
+
 public class WasapiDownloader
 {
-    public static void main(String[] args) {
-        System.out.println("I am WasapiDownloader. Hear me roar. Woo.");
+  public static final String SETTINGS_FILE_LOCATION = "config/settings.properties";
+  public Properties settings = null;
+
+  public WasapiDownloader() {
+    loadPropertiesFile();
+  }
+
+  private void loadPropertiesFile() {
+    if (settings == null)
+    {
+      InputStream input = null;
+      try {
+        input = new FileInputStream(SETTINGS_FILE_LOCATION);
+        settings = new Properties();
+        settings.load(input);
+      } catch (IOException e) {
```

Comment:
discussed w/ naomi, we both felt OK just letting the exception bubble up.  simplifies the code, and makes it more reusable by other Java code.  getting rid of the explicit `System.exit` might make it slightly less friendly to command-line script callers, but we can always add some optional error handling into the `main` method or something if that actually becomes an issue.  i think our assumption is that this will mostly be invoked by humans who will be manually reading the command output.